attributes/default.rb

```diff
@@ -40,7 +40,7 @@
 default[:valhalla][:routing_service_layers]                      = 'YOUR_LAYER_IDS'
 default[:valhalla][:routing_service_elb]                         = 'YOUR_ELB_NAME'
 default[:valhalla][:routing_service_recipes]                     = 'valhalla::get_routing_tiles'
-default[:valhalla][:min_routing_service_update_instances]        = 2
+default[:valhalla][:min_layers_instances]                        = '1'
```

Comment:
yeah its kind of not the best but, it ends up being the easiest. basically this ends up landing in a python script. so i treat it like a string here and then break it up into an array of integers in python. the other option would have been to put straight up python syntax inside of this string like:

    '[1,3,4]'

but i thought that looked less nice and was very python specific